.goreleaser.yml

```diff
@@ -22,11 +22,11 @@ checksum:
 snapshot:
   name_template: "{{ .Tag }}-next"
 
-blobs:
-  - provider: s3
-    bucket: downloads.scylladb.com
-    region: us-east-1
-    folder: "gemini/{{.Version}}"
+# blobs:
```

Comment:
If we are only gonna use docker to run Gemini, yes this can go away.

Anyhow it should work and have access to secrets, if run via tag creation, but won't be working during PRs

So it can be optional bases if it's triggered by tag or by PR

Reply:
Yes it can stay, but it blocks us from releasing gemini, there are no credentials to AWS and I'm not sure how important `gemini` on S3 is since in SCT we are using Docker images. I can contact Yaron to setup S3 credentials and have the `gemini` released on S3 as well